packages/curriculum-helpers/README.md

```diff
@@ -58,3 +58,4 @@ Logs
   ]
 }
 ```
+
```

Comment:
will I be able to publish a new patch for `curriculum-helpers` if I only edited the README?